tests/tool/tests/test_mkappimage.py

```diff
@@ -29,12 +35,24 @@ def fake_make_squashfs(_params: mkappimage.AppDirParams, _mksquashfs_params: Ite
 
 
 @mock.patch("appimage.private.tool.mkappimage.make_squashfs", fake_make_squashfs)
-def test_make_appimage() -> None:
-    """Test that the AppImage is created by concatenating the runtime and the squashfs."""
-    params = mkappimage.AppDirParams(Path(), Path(), Path(), Path(), Path())
+def test_make_appimage_x86_64() -> None:
+    """Test that the AppImage is created by concatenating the x86_64 runtime and the squashfs."""
+    runtime_path = mkappimage._get_path_or_raise("rules_appimage/tests/tool/tests/appimage_runtime_x86_64")
```

Comment:
I think it would be better to use the native arch version here like in the other test. That simplifies the BUILD dependencies.

Reply:
That's the test below :) 

Let me know if you'd prefer I delete this test

Reply:
Oh I see :D yes let's delete this one, I'm not sure it adds much value then